src/abstractdataframe/reshape.jl

```diff
@@ -259,6 +260,14 @@ Row and column keys are ordered in the order of their first appearance.
   time). Whether or not tasks are actually spawned and their number are
   determined automatically. Set to `false` if `combine` requires serial
   execution or is not thread-safe.
+- `sortrows`: the order of rows in the output table; all values accepted by
```

Comment:
```suggestion
- `sortrows`: the order of rows in the resulting table; all values accepted by
```

I prefer something else than "output", I find it a little ambiguous.